src/components/home.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { FaArrowAltCircleRight } from 'react-icons/fa';
+import { fetchCrypto } from '../redux/crypto/cryptoSlice';
+import Loading from './loads';
+
+const Home = () => {
+  const [search, setSearch] = useState('');
+  const coins = useSelector((state) => state.cryptoData);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchCrypto());
+  }, [dispatch]);
+  const handleSearch = (e) => {
+    setSearch(e.target.value);
+  };
+
+  const filteredCoins = coins.crypto.filter((coin) => (
+    coin.name.toLowerCase().includes(search.toLowerCase())
+  ));
+  if (coins.isLoding || coins.isEror) {
+    return <Loading />;
+  }
+
+  return (
+    <header className="coin-container">
+      <div className="home-serch">
+        <input
+          type="text"
+          value={search}
+          placeholder="Search Coin..."
+          onChange={handleSearch}
+        />
+      </div>
+      <div className="home-list">
+        {filteredCoins.length === 0 ? (
+          <h1>No related coins found!</h1>
+        ) : (
+          filteredCoins.map((coin) => (
+            <div key={coin.id} className="container-card">
+              <img src={coin.icon} alt={coin.name} />
+              <div className="container-card-info">
+                <h1>{coin.name}</h1>
+                <p>
+                  Price: $
+                  {coin.price.toFixed(2)}
+                </p>
+              </div>
+              <NavLink to={`/details/${coin.id}`}>
+                <FaArrowAltCircleRight className="link" />
+              </NavLink>
+            </div>
+          ))
```

Comment:
- [ ] Great job working on Homepage so far but I think it would be great if you just make the layout the same as the given design since here you have your Home page the items is under each others but it is different so you should make it like this so I highly recommend to update the style a little bit here to match the given one 🤩✅🔥💪🏻



### You can easily use grid and grid-template-columns property 🔥✅💯
- Your design
![image](https://user-images.githubusercontent.com/58553711/233093310-dded835a-2a07-4219-8071-de87591e1343.png)


## It should be beside each others 🙏🏻
- Design
![image](https://user-images.githubusercontent.com/58553711/233093576-563c9194-dd2a-4299-88ce-74799b62dce2.png)